src/main/java/Duke.java

```diff
@@ -1,10 +1,61 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
 public class Duke {
+    public static void printList(List<Task> list) {
+        for (int i = 0; i < list.size(); i++) {
+            System.out.println((i+1)+"."+list.get(i).getDoneStatus()+" "+list.get(i).getDescription());
+        }
+    }
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String lineBreak = "----------------------------------------------------------";
```

Comment:
Your `lineBreak`, `idle` and `happy` Strings should be set as constants. Declare them ```private final```.